spec/core/data_structures.md

```diff
@@ -258,14 +285,17 @@ Valid Duplicate Vote Evidence must adhere to the following rules:
 
```

Comment:
"Evidence must not have expired" does not really belong here. Or it should be explained differently: The first 4 bullet points are independent of time, that is, I can still validate them in two years from now. The last one depends on time, and is not a validating rule, but rather a guarantee of Tendermint at the time when Evidence is passed to the application.

Reply:
I can reword this to be specific for block inclusion.